app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,23 @@
+class SessionsController < ApplicationController
+    get '/sessions/new' do
+      erb :"sessions/new.html"
+    end
+
+    post '/sessions/login' do
+        user = User.find_by(email: params[:email])
+        if user && user.authenticate(params[:password])
+            session[:user_id] = user.id
```

Comment:
Rather than storing the email they put in the form in the `session`, you can pass it into the `erb` template via. `locals`